source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -76,6 +76,9 @@ class ZipkinSpan : public Tracing::Span {
   Zipkin::Span& span() { return span_; }
 
 private:
+  ZipkinSpan();
```

Comment:
use https://github.com/envoyproxy/envoy/blob/master/source/common/common/non_copyable.h (inherit from that)